src/Localization/en/SqlX.json

```diff
@@ -40,7 +40,7 @@
     "CosmosD4Details": "General Purpose Cosmos Compute with 4 vCPUs, 16 GB Memory",
     "CosmosD8Details": "General Purpose Cosmos Compute with 8 vCPUs, 32 GB Memory",
     "CosmosD16Details": "General Purpose Cosmos Compute with 16 vCPUs, 64 GB Memory",
-    "Cost": "Cost",
+    "ApproximateCost": "Approximate Cost Per Hour",
```

Comment:
Could we mention somewhere that the cost is in USD? Right now, we just have the $ in the cost estimate which could be ambiguous.

Reply:
Made it US $, does that work?

Reply:
yeah that works!